src/plugins/discover/public/application/main/components/field_stats_table/field_stats_tab.tsx

```diff
@@ -17,10 +18,12 @@ export const FieldStatisticsTab: React.FC<Omit<FieldStatisticsTableProps, 'query
     const querySubscriberResult = useQuerySubscriber({
       data: services.data,
     });
+    const savedSearch = useSavedSearch();
```

Comment:
why not using the state container here ... it's not available in the saved search embeddable